tests/quotes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -64,6 +64,10 @@ exports[`strings.js 1`] = `
 // Unnecessary escapes.
 "\\'"
 '\\"'
+"\\a"
+'\\a'
+"hol\\a"
+'hol\\a'
```

Comment:
Could you beef up your test plan quite dramatically? You are introducing a very error prone piece of logic in this pull request, it would give me confidence if you added tests for all the possible combinations you are adding.